Task2/Project/Shapes/Commands/Pentagon/AddPentagon.cs

```diff
@@ -1,23 +1,113 @@
-﻿namespace Shapes.Commands.Pentagon
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Shapes.Commands.Pentagon
 {
+    /// <summary>
+    /// Represents command of adding <see cref="Models.Pentagon"/>.
+    /// </summary>
     public class AddPentagon : Interfaces.ICommand
     {
-        public string Name
+        // CONST
+        private const int COUNT_VERTEX = 5;

```

Comment:
are Pentagon has these property, use it instead

Reply:
Done